### ISSUE TITLE
Enhancement: use Buttons with help menu

### ISSUE THREAD

**Issue by foxcris** [foxcris/appdaemon-telegrambot#6]
I think it will look better if the menu show always with buttons and nice graphic,
I manage to display the all menu with buttons but got stuck with how to relate each button to method...
see:

````
  if command == "/help":
        
            msg = "The following commands are available:\n"
           keyboard = []
           assign_key = []
## Menu display as button - but not sure how to assign menu- maybe break menu as separate function ??
           keyboard.append([("Menu",f"/menu")])
           for command in self._commanddict:
                desc = f'{self._commanddict.get(command).get("desc")}'
                method_str = f'/self.{self._commanddict.get(command).get("method").__name__}'
##Not sure how to assign the method to button 
                keyboard.append([(desc,method_str)])                
           str_keyboard = ', '.join(map(str, keyboard))
           self._call_service(msg,user_id,keyboard)
  ..... 

    def _call_service (self, msg ,target_id , keyboard = []):
        self.log(keyboard)
        self.call_service(
            "telegram_bot/send_message",
            target=target_id,
            message=self._escape_markdown(msg),
            inline_keyboard=keyboard
        )  
````
wish i could be more helpful with coding and actually be your right hand man here as i really like this project and find it very useful .
i will do my best but it's taking time .

_Originally posted by @avico78 in https://github.com/foxcris/appdaemon-telegrambot/issues/2#issuecomment-623028037_

**Comment by foxcris** [comment]
Implemented in Release [v1.1](https://github.com/foxcris/appdaemon-telegrambot/releases/tag/v1.1)

**Comment by avico78** [comment]
> Implemented in Release [v1.1](https://github.com/foxcris/appdaemon-telegrambot/releases/tag/v1.1)

Suggestion:

Change button name from "/<some_functionality>" to "some functionality "
i.e  "/state_sensor" --> "state sensor",
is there option to set the font to bold  ?
